### PR TITLE
Using importlib for version in sphinx

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Hello, 
This PR aims to implement #1045.
Very similar implementation has been done. Also, fixes some indentation/whitespace issues arising from linting. Inspired from [here](https://github.com/astropy/astropy/pull/11091/files) :smile:

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -89,50 +95,53 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
+project_ver = version(project)
+
 # The short X.Y version.
-version = '0.15'
+version = '.'.join(project_ver.split('.'[:2]))
```
Comment (astrojuanlu) [inline docs/source/conf.py:101]: Something is not right in this line.

**Review by astrojuanlu — CHANGES_REQUESTED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (DhruvSondhi) [reply docs/source/conf.py:101]: Got it :)

Comment (DhruvSondhi) [reply docs/source/conf.py]: Fixed the issue :smile:

File: tests/tests_twobody/test_propagation.py [hunk 2]
```diff
@@ -39,7 +39,7 @@
 from poliastro.util import norm
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
```
Comment (astrojuanlu) [inline tests/tests_twobody/test_propagation.py:42]: Excellent!

**Review by astrojuanlu — APPROVED** [review]
This is good to go, thanks a lot @DhruvSondhi!

File: tests/tests_twobody/test_propagation.py (hunk 2 above)
Comment (DhruvSondhi) [reply tests/tests_twobody/test_propagation.py:42]: Thank you very much :)

Comment (DhruvSondhi) [reply tests/tests_twobody/test_propagation.py:42]: Just a question... Should I squash the commits or is it alright? :thinking:

Comment (astrojuanlu) [reply tests/tests_twobody/test_propagation.py:42]: This is perfect! Each commit for a different thing

**Comment by astrojuanlu** [comment]
Lovely green color ✅  Merging!

**Pull request merged**